pkgs/tools/system/dc3dd/default.nix

```
@@ -0,0 +1,25 @@
+{ lib, stdenv, fetchzip }:
+
+stdenv.mkDerivation rec {
+  pname = "dc3dd";
+  version = "7.3.0";
+
+  src = fetchzip {
+    url = "mirror://sourceforge/dc3dd/dc3dd-${version}.zip";
+    sha256 = "pYr9A9444yC0nYDOf9oDGqgzgQ5Qgtt+J4M7J35HpQg=";
+  };
+  preConfigure = "chmod +x ./configure";
+  configureFlags = [
+  "--infodir=$info/share/info"
+  ] ++ (if stdenv.isDarwin then ["gl_cv_func_stpncpy=yes"] else []);
+  meta = with lib; {
```

My personal view is that a good readability is preferred over keeping the file as short as possible.

```suggestion
  preConfigure = "chmod +x ./configure";

  configureFlags = [
    "--infodir=$info/share/info"
  ] ++ (if stdenv.isDarwin then [
    "gl_cv_func_stpncpy=yes"
  ] else []);

  meta = with lib; {
```